developer/contributing.md

```diff
@@ -31,6 +31,84 @@ Start small and open up a dialogue with us. This will help to get your contribut
 -   Contributors should review the CLA.
 -   Code reviewed before merge acceptance.
 
+### Pull request template
+
+```
+Resolves #issueNumber
+Impact breaking|minor|bug|style|refactor
+
+## Issue
+
+Description of the issue this PR is solving, why it's happening and how it can be reproduced. This may differ from the original ticket as you now have more information at your disposal.
+
+## Solution
+
+Summarize the your solution to the problem, as well as any things you may have tried along the way that may not have worked.
```

Comment:
I think this part is really important, but it's a hard to follow (`the your`, `any things`)
What do you think about:
`Summarize your solution to the problem. Please include short descriptions of any solutions you tested before arriving at your final solution. This will help reviewers know why you decided to solve this problem in this particular way and will speed up the review process.`

Reply:
👍